imports/server/getTeamName.ts

```diff
@@ -1,7 +1,7 @@
 import Settings from '../lib/models/Settings';
 
-function getTeamName(): string {
-  const teamNameObj = Settings.findOne({ name: 'teamname' });
+async function getTeamName(): Promise<string> {
```

Comment:
You appear to have not updated any of the call sites of `getTeamName()` in `imports/server/gdrive.ts` nor `imports/server/methods/updatePuzzle.ts` so I think those will wind up with `[object Promise]` instead of the team name.

I'm guessing maybe template literals are a blind spot for the unawaited-promises lint?  Can we disallow promises in template literals, since I can't imagine us ever wanting to print `[object Promise]`?

Reply:
Good catch. I found a linter ([@typescript-eslint/no-base-to-string](https://typescript-eslint.io/rules/no-base-to-string/)) that catches this. Fortunately, it doesn't seem like we had any preexisting failures of that linter, so I feel relatively good about its precision. Should be fixed now.

Reply:
Excellent, thanks!